source/features/index.tsx

```diff
@@ -1,9 +1,11 @@
 import React from 'dom-chef';
 import select from 'select-dom';
 import domLoaded from 'dom-loaded';
+import cache from 'webext-storage-cache';
```

Comment:
I try to align the "from" so the module names are easier to follow/read
```suggestion
import React from 'dom-chef';
import cache from 'webext-storage-cache';
import select from 'select-dom';
import domLoaded from 'dom-loaded';
```